### ISSUE TITLE
Remove Compiled classes from the distributed jar

### ISSUE THREAD

**Issue by obohrer** [fzakaria/slf4j-timbre#27]
Hi,

Thank you for building this adaptor :)

We started to encounter some issues related to dependencies and it seems slf4j-timbre is released with compiled classes in the JAR (~278Kb).
```
  0 Tue Jun 27 19:37:36 BST 2017 slf4j_timbre/
  1332 Tue Jun 27 19:37:36 BST 2017 slf4j_timbre/adapter$_isErrorEnabled.class
   966 Tue Jun 27 19:37:36 BST 2017 slf4j_timbre/adapter$define_methods$iter__2498__2504.class
   903 Tue Jun 27 19:37:36 BST 2017 slf4j_timbre/adapter$_error_Marker_String_Object$fn__2574.class
   815 Tue Jun 27 19:37:36 BST 2017 slf4j_timbre/adapter$_trace_String_Object$fn__3242.class
  4701 Tue Jun 27 19:37:36 BST 2017 slf4j_timbre/adapter$_debug_Marker_String_Object.class
  3985 Tue Jun 27 19:37:36 BST 2017 slf4j_timbre/adapter$_warn_Marker_String.class
   822 Tue Jun 27 19:37:36 BST 2017 slf4j_timbre/adapter$_error_Marker_String_Object$fn__2576.class
  1339 Tue Jun 27 19:37:36 BST 2017 slf4j_timbre/static_logger_binder$_init.class
  1323 Tue Jun 27 19:37:36 BST 2017 slf4j_timbre/static_marker_binder$_init.class
   830 Tue Jun 27 19:37:36 BST 2017 slf4j_timbre/adapter$_error_Marker_String_Object_LT__GT_$fn__2654.class
```

It can causes compatibility issues and weird behaviour ( possibly https://dev.clojure.org/jira/browse/CLJ-1886 ?)

Would it be possible to release slf4j-timbre without the compiled java classes, just the source code ?

**Comment by rufoa** [comment]
I thought we managed to fix this in 0.3.5 :disappointed: (see the end of issue #8)

We rely heavily on AOT so I don't think we can remove any more of the compiled classes from the jar unfortunately.

Do you have an example project I can look at that exhibits the problem you're seeing please?

If it's private please post the output of `lein deps :tree`

**Comment by rufoa** [comment]
Closing this issue because I can't reproduce it with the info provided.

If @obohrer or anyone else is still having this problem and can provide a failing test case for me to investigate, please feel free to reopen. :+1:

Thanks
